corim/unsignedcorim_test.go

```diff
@@ -72,9 +67,9 @@ func TestUnsignedCorim_AddComid_and_marshal(t *testing.T) {
 
 	fmt.Printf("CBOR: %x", actual)
 
-	expected := testGoodUnsignedCorim
+	expected := testGoodUnsignedCorimCBOR
```

Comment:
For readers sake, now please refer to link, where these variables are taken from ?

Reply:
? we don't do that for any other commonly defined test variables, and this is something that is aready easily resolved by the IDE.